wagtailnhsukfrontend/templates/wagtailnhsukfrontend/care_card.html

```diff
@@ -1,6 +1,6 @@
 <div class="nhsuk-care-card nhsuk-care-card--{{ value.type }}">
   <div class="nhsuk-care-card__heading-container">
-    <h3 class="nhsuk-care-card__heading">
+    <h{{ heading_level }} class="nhsuk-care-card__heading">
```

Comment:
The closing tag for this heading level needs to be `<h{{ heading_level }}>`, not `</h3>`

Reply:
Thanks Adam

Reply:
@Ross-Clark not sure if you forgot to update the code or were just thanking me, but the code is still `</h3>` 😄

Reply:
I blame the wifi